ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/WizardNewNgProjectCreationPage.java

```diff
@@ -22,6 +24,11 @@
  */
 public class WizardNewNgProjectCreationPage extends WizardNewTypeScriptProjectCreationPage {
 
+	public static final String projectNameRegexp = "^[a-zA-Z][.0-9a-zA-Z]*(-[.0-9a-zA-Z]*)*$";
+	public static final String[] unsupportedProjectNames = new String[] { "test", "ember", "ember-cli", "vendor", "app" };
+
+	private static final Status ERROR_STATUS = new Status(IStatus.ERROR, AngularCLIPlugin.PLUGIN_ID, AngularCLIMessages.NewAngular2ProjectWizard_invalidProjectName);

```

Comment:
I think we should use IStatus.WARN in order to avoid blocking wizard, if convention name change in future version of cli